diesel_compile_tests/tests/fail/array_expressions_must_be_same_type.stderr

```diff
@@ -106,7 +106,7 @@ error[E0277]: the trait bound `{integer}: diesel::SelectableExpression<()>` is n
               <(A, B) as diesel::SelectableExpression<QS>>
               <(A, B, C) as diesel::SelectableExpression<QS>>
               <(A, B, C, D) as diesel::SelectableExpression<QS>>
-            and 108 others
```

Comment:
@pksunkara It's not clear for me why this change is happening here. The old `compile_tests` setup didn't have this "problem".